### PR TITLE
assisting packages and docs

### PR DESCRIPTION
**Pull request by GantMan**
So I came back to try to contribute to Reactotron, and keep in mind I'm like "ER MA GERD WUT?" so please forgive me if any of this is wrong/strange.

So there were TONS of deps that were missing for me.  Every time I ran a command from the doc, it was like "BWAHAHAHA, I aint doin' shit!"  It made me feel kinda bad about life.  So I went to the psychiatrist, and she said, I needed to go through reconciliation with a moment of truth or I'd keep wetting the bed... long story longer... I decided to make this pull request, knowing I'm probably adding deps that couldn't really be missing.  But in my own strange 6th sense kind of way they really are.  Anyhow, this is how I got it commands to work on my machine.

🦄

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ npm install
 npm run welcome
 ```
 
+You may need to install `rollup` globally.
```
Comment (skellock) [inline docs/contributing.md]: You don't.  Which package was causing this?

**Comment by skellock** [comment]
Let's chat in person tomorrow about what was going wrong.  None of these changes should be required to build.

**Comment by GantMan** [comment]
It's outside the history of my terminal now.  Bc I'm just typing up a storm.  But it didn't work.   Installing these made it work *shrug*

File: docs/contributing.md (hunk 1 above)
Comment (GantMan) [inline docs/contributing.md]: will round it up with you later and see.

**Comment by skellock** [comment]
Alright. I'll close this PR up though. Root dependencies aren't the right spot for these dependencies.

**Pull request closed without merging**